src/components/loginForm.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react"
+import { makeStyles } from "@material-ui/core/styles"
+import Button from "@material-ui/core/Button"
```

Comment:
Let's compress all these `@material-ui/core` imports like in the `header.tsx` and `layout.tsx` components!

Reply:
completed!